### ISSUE TITLE
Use dns.setServers from nodej's dns from v0.11

### ISSUE THREAD

**Issue by skepticfx** [skepticfx/subquest#4]
Switch back to the require('dns') which will support serServers and we no longer rely on native-dns.

**Comment by b4dnewz** [comment]
I will probably do this as soon as possible, to make the package use only node __core modules__, It's still not clear (to me) which of the [dns methods](https://nodejs.org/api/dns.html) is the good to archive the same results.

**Comment by skepticfx** [comment]
@b4dnewz I would use the simpler `.resolve` method: https://nodejs.org/api/dns.html#dns_dns_resolve_hostname_rrtype_callback with the default 'A' record.

**Comment by b4dnewz** [comment]
Ok I've worked on this, It was quite painful, I had to change a lot of code but now is much simpler and faster, too.

I need to write some stuff into the README file since I get rid of event emitters too in favor of simple callbacks and the structure is more readable, tomorrow afternoon when I've time to finish I'll push this PR.

Also I've another proposal for this repo, we should split the cli-code in a separate repository, which can be also optimized. I think this a needed change because the two things are separate and usually peoples who use it as a module will never use the cli code at all.

I open a new issue to keep track of this, are you ok with this?
